core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -49,19 +50,25 @@ class HttpExecutionServiceDAO(executionServiceURL: String, override val workbenc
       case wdl: WdlSource => Multipart.FormData.BodyPart("workflowSource", wdl.source)
     }
 
+    val inputsJsonArray = inputs.mkString("[", ",", "]")
+
+    // Map over inputs to make this list the same size, but ihe inputs themselves are ignored.
+    val requestedWorkflowIdsJsonArray = inputs.map(_ => s""""${UUID.randomUUID().toString}""""  ).mkString("[", ",", "]")
```

Comment:
Since we're now submitting known workflow IDs to Rawls, does it make sense to pre-populate database entries into the Rawls workflows table?

The code as-is will work fine, but it seems a bit odd that Rawls will generate the ID, send it to Cromwell, hear it echoed back, and then finally register it as a monitored workflow.

Reply:
I think... potentially yes that would be nice, with a few caveats -

1. I think users are used to thinking "workflow ID means it's been started" which we'd have to re-teach
2. The UI thinks "workflow ID means I can show the Job Manager/Workflow Dashboard link", which we'd want to change
3. More difficult than I was hoping for in this PR/

I can make a follow up ticket if you think it's worth it though?

Reply:
OK so long as it's what you have in mind I don't see a problem with it, and no need to add followup work (that we likely won't ever do)